website/source/docs/providers/google/d/datasource_compute_network.html.markdown

```diff
@@ -0,0 +1,50 @@
+---
+layout: "google"
+page_title: "Google: google_compute_network"
+sidebar_current: "docs-google-datasource-compute-etwork"
+description: |-
+  Get a network within GCE.
+---
+
+# google\_compute\_network
+
+Get a network within GCE from his name.
+
+## Example Usage
+
+```js
+datasource "google_compute_network" "my-network" {
+  name          = "default-us-east1"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the network.
+    
+
+- - -
+
+* `project` - (Optional) The project in which the resource belongs. If it
+    is not provided, the provider project is used.
+
+## Attributes Reference
+
+In addition to the arguments listed above, the following computed attributes are
```

Comment:
The other gcp data sources docs seem to be using this phrasing: "The following attributes are exported:"